client/components/social-buttons/google.js

```diff
@@ -158,9 +176,11 @@ class GoogleLoginButton extends Component {
 
 		const { responseHandler } = this.props;
 
+		this.setState( { isClicked: null } );
+
 		// Options are documented here:
 		// https://developers.google.com/api-client-library/javascript/reference/referencedocs#gapiauth2signinoptions
-		window.gapi.auth2
+		window?.gapi.auth2
```

Comment:
I think this is probably not what you intended. This only works as intended if `window` is defined but nullish, which doesn't happen neither on the server, nor on the client.

I think that it's more likely that you're checking for whether `window.gapi` and not `window` is defined, however. For that, you should do `window.gapi?.auth2` instead.